CHANGELOG.md

```diff
@@ -4,16 +4,15 @@ x.x.x Release notes (yyyy-MM-dd)
 * None.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* Fixed a bug in use of dangling accessors after integrating a changeset. The bug could lead to fatal crashes. ([RSYNC-71](https://jira.mongodb.org/browse/RSYNC-71), since v2.0.0)
```

Comment:
"dangling accessors" is about implementation. Let's stick to what users care about. (see Cocoa release note)